app/scripts/lib/account-tracker.js

```diff
@@ -225,29 +222,28 @@ class AccountTracker {
     this.store.updateState({ accounts })
   }
 
-  // /**
-  //  * Updates current address balances from balanceChecker deployed contract instance
-  //  * @param {*} addresses
-  //  * @param {*} deployedContractAddress
-  //  */
-  // async _updateAccountsViaBalanceChecker (addresses, deployedContractAddress) {
-  //   const accounts = this.store.getState().accounts
-  //   this.web3.setProvider(this._provider)
-  //   const ethContract = new this.web3.eth.Contract(SINGLE_CALL_BALANCES_ABI, deployedContractAddress)
-  //   const ethBalance = ['0x0']
-
-  //   ethContract.methods.balances(addresses, ethBalance).call(null, (error, result) => {
-  //     if (error) {
-  //       log.warn(`MetaMask - Account Tracker single call balance fetch failed`, error)
-  //       return Promise.all(addresses.map(this._updateAccount.bind(this)))
-  //     }
-  //     addresses.forEach((address, index) => {
-  //       const balance = bnToHex(result[index])
-  //       accounts[address] = { address, balance }
-  //     })
-  //     this.store.updateState({ accounts })
-  //   })
-  // }
+  /**
```

Comment:
Why are we re-enabling this / Why was this disabled previously? Was this a web3.js bug that is being fixed here?

Reply:
Correct; we had to use the fallback version of account tracking (not the smart contract) due to web3 issues. With this, it'll work again.